src/promptflow/promptflow/azure/_pf_client.py

```diff
@@ -60,7 +61,10 @@ def __init__(
             workspace_name=workspace_name,
             **kwargs,
         )
-        workspace = self._ml_client.workspaces.get(name=self._ml_client._operation_scope.workspace_name)
+        try:
+            workspace = self._ml_client.workspaces.get(name=self._ml_client._operation_scope.workspace_name)
+        except Exception as e:
+            raise UserErrorException(message=str(e), error=e)
```

Comment:
SDKErrror?

Reply:
If I change it to `SDKError`, then `error type` will be `SDKError `instead of `ClientAuthenticationError`, but I think `error type` should display `ClientAuthenticationError `better? So I use `UserErrorException`, which will use the value of error parameter when printing `error type`

![image](https://github.com/microsoft/promptflow/assets/23182548/d40d023e-5507-4296-a5df-5b195f1175a6)

Reply:
According to offline discussions, we will further optimize the details for the next PR, and merge the current PR first